smile_net.py

```diff
@@ -0,0 +1,22 @@
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class Net(nn.Module):
+  def __init__(self):
+    super(Net, self).__init__()
+    self.conv1 = nn.Conv2d(1, 10, kernel_size=5)
+    self.conv2 = nn.Conv2d(10, 20, kernel_size=5)
+    self.conv2_drop = nn.Dropout2d()
+    self.fc1 = nn.Linear(3380, 50)
```

Comment:
what are these numbers?
if they are consts or a function of batch_size - please give them names